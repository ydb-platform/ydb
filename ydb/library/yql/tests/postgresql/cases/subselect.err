<sql-statement>
--
-- SUBSELECT
--
SELECT 1 AS one WHERE 1 IN (SELECT 1);
</sql-statement>
-stdin-:<main>: Fatal: Optimization

    -stdin-:<main>:4:25: Fatal: ydb/library/yql/core/common_opt/yql_co_pgselect.cpp:2852  JoinOuter(): requirement outerInputs.size() == finalExtTypes->Tail().ChildrenSize() failed
    	SELECT 1 AS one WHERE 1 IN (SELECT 1);
	                        ^
    -stdin-:<main>:1:1: Fatal: ydb/library/yql/core/common_opt/yql_co_pgselect.cpp:640  RewriteSubLinksPartial(): requirement status.Level != IGraphTransformer::TStatus::Error failed
    	--
	^
<sql-statement>
SELECT 1 AS zero WHERE 1 NOT IN (SELECT 1);
</sql-statement>
-stdin-:<main>: Fatal: Optimization

    -stdin-:<main>:1:26: Fatal: ydb/library/yql/core/common_opt/yql_co_pgselect.cpp:2852  JoinOuter(): requirement outerInputs.size() == finalExtTypes->Tail().ChildrenSize() failed
    	SELECT 1 AS zero WHERE 1 NOT IN (SELECT 1);
	                         ^
    -stdin-:<main>:1:1: Fatal: ydb/library/yql/core/common_opt/yql_co_pgselect.cpp:640  RewriteSubLinksPartial(): requirement status.Level != IGraphTransformer::TStatus::Error failed
    	SELECT 1 AS zero WHERE 1 NOT IN (SELECT 1);
	^
<sql-statement>
SELECT 1 AS zero WHERE 1 IN (SELECT 2);
</sql-statement>
-stdin-:<main>: Fatal: Optimization

    -stdin-:<main>:1:26: Fatal: ydb/library/yql/core/common_opt/yql_co_pgselect.cpp:2852  JoinOuter(): requirement outerInputs.size() == finalExtTypes->Tail().ChildrenSize() failed
    	SELECT 1 AS zero WHERE 1 IN (SELECT 2);
	                         ^
    -stdin-:<main>:1:1: Fatal: ydb/library/yql/core/common_opt/yql_co_pgselect.cpp:640  RewriteSubLinksPartial(): requirement status.Level != IGraphTransformer::TStatus::Error failed
    	SELECT 1 AS zero WHERE 1 IN (SELECT 2);
	^
<sql-statement>
-- Check grammar's handling of extra parens in assorted contexts
SELECT * FROM (SELECT 1 AS x) ss;
</sql-statement>
<sql-statement>
SELECT * FROM ((SELECT 1 AS x)) ss;
</sql-statement>
<sql-statement>
(SELECT 2) UNION SELECT 2;
</sql-statement>
NYql::TAggregateExpander::ExpandAggApply(TIntrusivePtr<NYql::TExprNode, TDefaultIntrusivePtrOps<NYql::TExprNode> > const&) at /-S/ydb/library/yql/core/yql_aggregate_expander.cpp:119:51
?? at ??:0:0
