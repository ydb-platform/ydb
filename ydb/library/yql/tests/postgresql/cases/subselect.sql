-- Check grammar's handling of extra parens in assorted contexts
SELECT * FROM (SELECT 1 AS x) ss;
SELECT * FROM ((SELECT 1 AS x)) ss;
