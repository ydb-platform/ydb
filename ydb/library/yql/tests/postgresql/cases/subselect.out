-- Check grammar's handling of extra parens in assorted contexts
SELECT * FROM (SELECT 1 AS x) ss;
 x 
---
 1
(1 row)

SELECT * FROM ((SELECT 1 AS x)) ss;
 x 
---
 1
(1 row)

